src/status_im/ui/screens/chat/styles/message/message.cljs

```diff
@@ -93,6 +93,10 @@
     {:align-self    :flex-start
      :padding-left  (if platform/desktop? 24 8)}))
 
+(def message-author-touchable
+  {:margin-left        12
+   :padding-horizontal 2})
```

Comment:
@Ferossgp hey, my mistake. it should be vertical here if needed at all (text line height is 18, touchable is 22 so 2px padding would center it inside the touchable)